MapColornamestoPairnumber.c

```diff
@@ -0,0 +1,13 @@
+#include "declare.h"
+#include <stdio.h>
+void MapColornamestoPairnumber()
+{
+  for (int i=1; i<=25 ; i++)
```

Comment:
25 is a magic number. this poses a problem if the minor colors and major colors are extended with additional colors. the probability of causing undetected problems is high as one might forget to update this number in case of extension of colors.